test-cases/longevity/longevity-mini-test-1h.yaml

```diff
@@ -1,5 +1,5 @@
-test_duration: 180
-stress_cmd: ["cassandra-stress write cl=QUORUM duration=90m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=50 -pop seq=1..10000000 -log interval=5"]
+test_duration: 1000
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=900m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=2 -pop seq=1..10000000 -log interval=5"]
```

Comment:
i guess you changed this just to debug purpose, so, once you are done, please, revert the changes in here